Dockerfile

```diff
@@ -1,12 +1,33 @@
-FROM codacy/ci-base:2018.08.3
+FROM codacy/ci-base:1.0.1 AS builder
+
+RUN apk add --update --no-cache --force-overwrite \
+    openssl openssl-dev crystal shards g++ gc-dev \
+    libc-dev libevent-dev libxml2-dev llvm llvm-dev \
+    llvm-libs llvm-static make pcre-dev readline-dev \
+    yaml-dev zlib-dev git
+
+RUN  git config --global user.email "team@codacy.com" && git config --global user.name "Codacy"
+
+RUN mkdir -p /workspace
+
+WORKDIR /workspace
+COPY ./ /workspace
+
+ENV ALPINE=1
+
+RUN make test build
+
+
+
+FROM codacy/ci-base:1.0.1
 
 LABEL maintainer="team@codacy.com"
 
-WORKDIR /scripts
-COPY src/ ./src/
-COPY get_next_version.sh .
+RUN apk add --update --no-cache --force-overwrite gc-dev pcre-dev libevent-dev git
+
+COPY --from=builder /workspace/bin/git-version /bin
 
-RUN mkdir /repo
+RUN mkdir -p /repo
 VOLUME /repo
 
-ENTRYPOINT ["./get_next_version.sh", "/repo"]
+CMD ["/bin/git-version", "--folder=/repo"]
```

Comment:
Because the container is supposed to run the same command every time, probably you could place the command in `ENTRYPOINT` and add the flags to `CMD`.

Reply:
not really picky on that, but `CMD` is easier to override and in this contxt doesn't make much difference